vowpalwabbit/core/src/example.cc

```diff
@@ -15,6 +15,58 @@
 #include <climits>
 #include <cstdint>
 
+void VW::del_example_namespace(VW::example& ec, VW::namespace_index ns, const features& fs)
```

Comment:
is this a function that should be exposed directly to users via the public API?

Reply:
You're right, this seems like a good candidate for the details namespace